src/modules/translation/languages/en.json

```
@@ -460,15 +460,19 @@
     },
     "deploy_world": {
       "title": "Publish to World",
-      "description": "Select the NAME that will define the URL:",
+      "description": "Select a Decentraland NAME or ENS Name that will define the URL",
```

This should still be `the` as it's taking about a specific name:
```suggestion
      "description": "Select the Decentraland NAME or ENS Name that will define the URL",
```